front/pages/w/[wId]/builder/labs/transcripts/index.tsx

```diff
@@ -0,0 +1,348 @@
+      >
+        <Page>
+          <Page.Header
+            title="Transcripts processing"
+            icon={ChatBubbleLeftRightIcon}
+            description="Receive meeting minutes summarized by email automatically. Works with Google Meet and Gong.io."
+          />
+          <Page.Layout direction="vertical">
+            <Page.SectionHeader title="1. Connect Google Drive" />
+            <Page.Layout direction="horizontal">
+              <Page.P>
+                Connect your personal Google Drive so Dust can access your
+                meeting transcripts.
+              </Page.P>
+
+              <Button
+                label={isGDriveConnected ? "Connected" : "Connect"}
+                size="sm"
+                icon={CloudArrowLeftRightIcon}
+                disabled={isLoading || isGDriveConnected}
+                onClick={async () => {
+                  await handleConnectTranscriptsSource();
+                }}
+              />
+            </Page.Layout>
+          </Page.Layout>
+          {!isLoading && isGDriveConnected && (
+            <>
+              <Page.Layout direction="vertical">
+                <Page.SectionHeader title="2. Assistant configuration" />
+                <Page.Layout direction="horizontal">
+                  <Page.P>
+                    The chosen assistant should be configured to summarize the
+                    transcripts in the way you want.{" "}
+                    <strong>Instruct it to answer in HTML format</strong> to
+                    receive a readable email.
+                  </Page.P>
+
+                  <AssistantPicker
+                    owner={owner}
+                    size="sm"
+                    onItemClick={(c) => {
+                      void handleSelectAssistant(c).then(() => {
+                        console.log("Selected assistant " + c.name);
+                      });
+                    }}
+                    assistants={agents}
+                    showBuilderButtons={false}
+                  />
+                  {assistantSelected && (
+                    <Page.P>
+                      <strong>@{assistantSelected.name}</strong>
+                    </Page.P>
+                  )}
+                </Page.Layout>
+              </Page.Layout>
+              <Page.Layout direction="vertical">
+                <Page.SectionHeader title="3. Email configuration" />
+                <Page.Layout direction="horizontal" gap="xl">
+                  <Page.P>
+                    By default, the assistant will send the summarized
+                    transcript to your email. You can chose a different email
+                    here.
+                  </Page.P>
+                  {/* <Input placeholder="" name="input" value={emailToNotify} onBlur={(e) => setEmailToNotify(e)} /> */}
+                  {/* Had to put a regular input there because <Input> doesn't like onBlur */}
+                  <input
```

Comment:
Is anyone sees a better option using the sparkle component to get onBlur() working I'm all ears

Reply:
cc @Duncid or maybe @philipperolet who worked a bit with input bars I believe?

Reply:
hey! we could add the onBlur to sparkle. Quick question though: are we using focus loss (e.g changing tab or temporary going to another element) as the trigger for validating an email? (vs e.g. a save button)
AFAICT it's rather uncommon UX. Maybe it's appropriate here, but what was the rationale?

Reply:
Maybe I just haven't use the right way we auto-save without having a save button? Do we have a specific way to do so that I missed? (e.g. in the Dust apps, when updating code we don't seem to have a Save button)

Reply:
As discussed IRL, a save button can do the job for the email field (this way sparkle input can be used direct) :+1: Thanks!

Reply:
Done